app/scripts/lib/setupWeb3.js

```diff
@@ -71,15 +71,17 @@ export default function setupWeb3(log) {
       for (const [key, path] of applyTrapKeys) {
         web3[topKey][key] = new Proxy(web3[topKey][key], {
           apply: (...params) => {
-            window.ethereum.request({
-              method: 'metamask_logInjectedWeb3Usage',
-              params: [
-                {
-                  action: 'apply',
-                  path,
-                },
-              ],
-            })
+            window.ethereum
+              .request({
+                method: 'metamask_logInjectedWeb3Usage',
+                params: [
+                  {
+                    action: 'apply',
+                    path,
+                  },
+                ],
+              })
+              .catch(() => undefined)
```

Comment:
The error should at least be logged, so we can see that it happened. This could result in a very frustrating debugging experience